lighthouse-core/report/html/report-styles.css

```diff
@@ -792,7 +792,32 @@
   background: var(--color-fail);
 }
 
-
+.icon-button {
+  display: inline-flex;
+  align-items: center;
+  border: solid 2px grey;
+  height: 50px;
+  cursor: pointer;
+  box-shadow: 1px 1px 3px #ccc;
+}
+.icon-button:hover {
+  background-color: var(--color-gray-200);
+}
+.icon-button__preview {
+  aspect-ratio: 1/1;
+  height: 100%;
+  background-size: cover;
+}
+.icon-button--treemap .icon-button__preview {
```

Comment:
instead of a unique dom element for the icon you could set it as bg image for the entire thing.. same approach we use in the `lh-tools__dropdown`

adding on, we do all sorts of things in there, like `text-indent` to make room for the icon, etc.. i could see us reusing a lot of the same styling technique.